.github/workflows/generate-assets.yaml

```diff
@@ -0,0 +1,32 @@
+name: Generate Assets
+
+on:
+  workflow_run:
+    workflows: [OPA Tests]
+    types:
+      - completed
```

Comment:
Just to inform myself: when is this going to happen? Not familiar with this trigger type from before.

Reply:
This trigger will only run this workflow if the OPA Tests workflow completes successfully. I did this because I wanted to make sure we didn't generate the zip file if tests failed.

Reply:
Cool 👍